app/views/ExposureHistory/HasBeenExposedDetail.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { StyleSheet, View } from 'react-native';
+import dayjs from 'dayjs';
+
+import { Typography } from '../../components/Typography';
+
+import {
+  Typography as TypographyStyles,
+  Outlines,
+  Colors,
+  Spacing,
+} from '../../styles';
+
+type Posix = number;
+
+interface Exposure {
+  date: Posix;
+  possibleExposureTimeInMin: number;
+  currentDailyReports: number;
+}
+
+interface HasBeenExposedDetailProps {
+  exposure: Exposure;
+}
+
+const HasBeenExposedDetail = ({
+  exposure: { date, possibleExposureTimeInMin, currentDailyReports },
+}: HasBeenExposedDetailProps): JSX.Element => {
+  const exposureDate = dayjs(date).format('dddd, MMM DD');
```

Comment:
all of this should be translated in a future commit, given the demo on is tomorrow, it makes sense to hard code the content.  We can take a pass at adding translations next week.